harbor/setup.py

```diff
@@ -18,7 +18,7 @@
     long_description = f.read()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base>=4.2.0'
+CHECKS_BASE_REQ = 'datadog-checks-base>=9.3.0'
```

Comment:
Why 9.3.0 and not 10.1.0?
https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/CHANGELOG.md#1010--2020-01-03